### PR TITLE
lesson 6 homework (unfinished)

### PR DESCRIPTION
**Pull request by AZhang2**
problem 3 code acts like third value in all dictionaries doesn't exist, plz help fix
PROBLEM HAS BEEN FIXED

### CODE REVIEW

**Comment by AZhang2** [comment]
stuff...

**Comment by Shuzhengz** [comment]
> problem 3 code acts like third value in all dictionaries doesn't exist, plz help fix

emmm can u include a screenshot or smth?
thx

File: homework/lesson6/Lesson6homework1Andre.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Words = ["hello", "world", "of", "tanks", "video", "games", "phantom", "forces", "stupid", "hackers"]
+user = input("Please input a word (no capitals): ")
+if (user in Words) is False:
+    Words.append(user)
```
Comment (LZDCA) [inline homework/lesson6/Lesson6homework1Andre.py:4]: This is a really easy way to get comparison done! A new learning for me.

File: homework/lesson6/Lesson6homework1Andre.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Words = ["hello", "world", "of", "tanks", "video", "games", "phantom", "forces", "stupid", "hackers"]
+user = input("Please input a word (no capitals): ")
+if (user in Words) is False:
```
Comment (LZDCA) [inline homework/lesson6/Lesson6homework1Andre.py:3]: This is a really easy way to get comparison done! A new learning for me.

File: homework/lesson6/Lesson 6 homework-Andre.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+
```
Comment (Shuzhengz) [inline homework/lesson6/Lesson 6 homework-Andre.py:2]: Emmm can u delete this file?

**Pull request merged**